test/ejabberd_tests/rebar.config

```diff
@@ -4,7 +4,7 @@
 {require_otp_vsn, "1[789]"}.
 
 {deps, [
-        {escalus, ".*", {git, "git://github.com/esl/escalus.git", "cf285c0"}},
+        {escalus, ".*", {git, "git://github.com/esl/escalus.git", {branch, "add-caps-to-stream-features"}}},
```

Comment:
Why don't we merge it to master in escalus first, and then keep using master here? Just to keep separation of concerns, so that Mongoose doesn't need to know about how Escalus development is organised.